### PR TITLE
Integrate Cloud Shell in release process 

### PR DESCRIPTION
**Pull request by Daniel-Sanche**
In prep for tomorrow's release

fixes https://github.com/GoogleCloudPlatform/stackdriver-sandbox/issues/226

- make-release.sh will now update the website cloud shell button to reference the new cloud shell image tag
- our release automation will build and push new images for the cloud shell image

### CODE REVIEW

File: .github/workflows/push-tags.yml [hunk 1]
```diff
@@ -35,3 +35,13 @@ jobs:
                        --tag=${{ steps.get_version.outputs.VERSION  }}
       env:
         PROJECT_ID: ${{ secrets.PROJECT_ID }}
+    - name: Push Cloud Shell Image to GCR
+      timeout-minutes: 20
+      run: |
+        set -x
+        gcloud auth configure-docker --quiet
+        IMAGE_PATH=gcr.io/$PROJECT_ID/cloudshell-image:${{ steps.get_version.outputs.VERSION }}
```
Comment (eaberman) [inline .github/workflows/push-tags.yml:43]: In this line, what does steps.get_version.outputs.VERSION correspond to?

**Comment by eaberman** [comment]
Is there a path to the cloudshell image within the repo that needs to be specified?

File: .github/workflows/push-tags.yml (hunk 1 above)
Comment (simonz130) [reply .github/workflows/push-tags.yml:43]: It's a reference to other place in this yaml doc. In this case we want to tag a cloudshell image with the same version tag as the result of get_version step (part of the same workflow).

**Comment by simonz130** [comment]
> Is there a path to the cloudshell image within the repo that needs to be specified?

It's specified here:
```
        IMAGE_PATH=gcr.io/$PROJECT_ID/cloudshell-image:${{ steps.get_version.outputs.VERSION }}

```

**Pull request merged**